sparkbench/ml/src/main/scala/com/intel/sparkbench/ml/PCADataGenerator.scala

```diff
@@ -20,11 +20,9 @@ package com.intel.hibench.sparkbench.ml
 import com.intel.hibench.sparkbench.common.IOCommon
 
 import scala.util.Random
-
```

Comment:
can we follow the spark import style https://github.com/databricks/scala-style-guide#imports?  So let's keep the line

Reply:
@carsonwang  Thanks, it's done.